### DISCUSSION TITLE
Removal of `replace_illegal`

### DISCUSSION THREAD

**Discussion by denizdogan** [sabnzbd/sabnzbd#2205] [General]
Is there any particular reason why `replace_illegal` is being removed?

**Comment by Safihre** [comment]
There are a number of (Special) settings which were once introduced, but not documented why exactly they are needed. If there's a good reason to keep them, we'll just keep them of course.
So that's why there's the question: why did you change it from the default (in this case, it's enabled by Default)?

**Reply by denizdogan** [reply]
I use it because I my configuration involves a shared drive which doesn't play too nicely with unusual characters.

Is the idea to _remove_ the functionality or remove the possibility to _disable_ it?

Thanks

**Reply by Safihre** [reply]
Indeed to remove the possibility to disable it.
Do you currently have it disabled (unchecked)? Because in that case the bad characters are not removed.

**Reply by denizdogan** [reply]
@Safihre No, it's enabled! I was just worried that the funcitonality was going to be removed.

**Reply by goodyear77** [reply]
I’ve tried finding this setting but to no avail, I would have guessed it would be under “switches”? I’m on 3.6.0 and getting this warning.

**Reply by Safihre** [reply]
As mentioned above, it's a Special setting.
